packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -23,7 +23,8 @@ picking multiple images on pre-iOS 14 devices.
 
 * Improves the documentation on handling MainActivity being killed by the Android OS.
 * Updates Android compileSdkVersion to 31.
-* Fix iOS RunnerUITests search paths.
+* Fixes iOS RunnerUITests search paths.
+* Fixes Activity leak.
```

Comment:
Version 0.8.4+5, which is now 5 versions old, did not include this fix.

Reply:
done.